net/http/impl/server.h

```diff
@@ -47,9 +47,9 @@ typedef std::vector<std::pair<std::string, std::string>> HTTPHeadersType;
 namespace {
 
 const char kCRLF[] = "\r\n";
-const size_t kCRLFLength = CompileTimeStringLength(kCRLF);
+const size_t kCRLFLength = strings::CompileTimeStringLength(kCRLF);
 const char kHeaderKeyValueSeparator[] = ": ";
```

Comment:
[According to the specs](http://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2), the space after the header separator is optional, and that space may be any linear whitespace character, not only `0x20` space:
> The field value MAY be preceded by any amount of LWS, though a single SP is preferred.
> ...
> `message-header = field-name ":" [ field-value ]`